geowave-analytics/src/main/java/mil/nga/giat/geowave/analytics/tools/PropertyManagement.java

```diff
@@ -399,21 +419,19 @@ public DistributableQuery getPropertyAsQuery(
 		return toArguments(GeoWaveRunnerArguments);
 	}
 
-	public void buildFromOptions(
+	public synchronized void buildFromOptions(
 			final CommandLine commandLine )
 			throws ParseException {
-		synchronized (MUTEX) {
-			for (final Option option : commandLine.getOptions()) {
-				if (!option.hasArg()) {
-					properties.put(
-							option.getLongOpt(),
-							true);
-				}
-				else {
-					properties.put(
-							option.getLongOpt(),
-							option.getValue());
-				}
+		for (final Option option : commandLine.getOptions()) {
+			if (!option.hasArg()) {
+				properties.put(
+						option.getLongOpt(),
+						true);
```

Comment:
Not sure what will happen here - not supposto have anything but string, string in a properties, but this snuck in using the put method.  Might implicitly get converted, but probably better to be explicit.